cds_migrator_kit/rdm/migration/transform/xml_processing/rules/base.py

```diff
@@ -184,7 +193,7 @@ def creators(self, key, value):
 
 
 @model.over("contributors", "^700__")
-@for_each_value
+# @for_each_value
```

Comment:
leftover?

Reply:
it should iterate no?

Reply:
No it shouldn’t because it is duplicating the decorator from above, so it creates list of lists

Reply:
minor: to be removed